### PR TITLE
Feature/admin view

### PR DESCRIPTION
**Pull request by LeeVoi**
Finished making the fxml and the controller for the admin main-view UI and controller

### CODE REVIEW

**Comment by patrikvalentiny** [comment]
I will comment on this tomorrow but I feel like this is very much not on point of what I would want as an admin, a good start but I would change it up

**Comment by LeeVoi** [comment]
worked on the logoutButton, fixed by Partik, this branch also contains the Admin-view FXML

File: src/main/java/dk/easv/Main.java [hunk 1]
```diff
@@ -15,8 +15,10 @@
 import java.util.Optional;
 
 public class Main extends Application {
+    private static Stage primaryStage;
```
Comment (patrikvalentiny) [inline src/main/java/dk/easv/Main.java]: no reason to have a static variable here

File: src/main/java/dk/easv/Main.java [hunk 2]
```diff
@@ -47,6 +49,9 @@ private static void showConfigNotFoundAlert() {
         }
     }
 
+    public static Stage getPrimaryStage() {
```
Comment (patrikvalentiny) [inline src/main/java/dk/easv/Main.java]: no need for this variable and method at all

**Review by patrikvalentiny — CHANGES_REQUESTED** [review]

**Review by patrikvalentiny — APPROVED** [review]

**Pull request merged**